src/azure-cli/azure/cli/command_modules/security/tests/latest/recordings/test_security_iot.yaml

```diff
@@ -68,7 +68,7 @@ interactions:
       User-Agent:
       - AZURECLI/2.33.0 azsdk-python-azure-mgmt-iothub/2.2.0 Python/3.8.3 (Windows-10-10.0.22000-SP0)
     method: PUT
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest.rg000001/providers/Microsoft.Devices/IotHubs/azurecli-hub000002?api-version=2021-07-02
```

Comment:
Please fix the test failure with new API version instead of manually replacing API verison in yaml files

Reply:
tracking issue here https://github.com/Azure/azure-cli/issues/24382

Reply:
@vilit1 I believe this issue will also affect users. Before you solve this issue, we can't merge this PR, right?

Reply:
@vilit1 Since we will launch the release of this sprint tomorrow, please resolve all the comments before tomorrow, otherwise this PR will not be able to catch up with the release of this sprint, and it can only be postponed to the next sprint **(12-06)**

Reply:
This will fix will be postponed until the fix is avaliable in the second week of november.

Reply:
Fixed!